tests/actions.go

```diff
@@ -1511,78 +1516,74 @@ func (oa *operatorActions) pdMembersReadyFn(tc *v1alpha1.TidbCluster) (bool, err
 	}
 
 	if tc.Status.PD.StatefulSet == nil {
-		log.Logf("tidbcluster: %s/%s .status.PD.StatefulSet is nil", ns, tcName)
+		log.Logf("TidbCluster: %q .status.PD.StatefulSet is nil", tcID)
 		return false, nil
 	}
 	failureCount := len(tc.Status.PD.FailureMembers)
 	replicas := tc.Spec.PD.Replicas + int32(failureCount)
 	if *pdSet.Spec.Replicas != replicas {
-		log.Logf("statefulset: %s/%s .spec.Replicas(%d) != %d",
-			ns, pdSetName, *pdSet.Spec.Replicas, replicas)
+		log.Logf("StatefulSet: %q .spec.Replicas(%d) != %d", pdStsID, *pdSet.Spec.Replicas, replicas)
 		return false, nil
 	}
 	if pdSet.Status.ReadyReplicas != tc.Spec.PD.Replicas {
-		log.Logf("statefulset: %s/%s .status.ReadyReplicas(%d) != %d",
-			ns, pdSetName, pdSet.Status.ReadyReplicas, tc.Spec.PD.Replicas)
+		log.Logf("StatefulSet: %q .status.ReadyReplicas(%d) != %d", pdStsID, pdSet.Status.ReadyReplicas, tc.Spec.PD.Replicas)
 		return false, nil
 	}
 	if len(tc.Status.PD.Members) != int(tc.Spec.PD.Replicas) {
-		log.Logf("tidbcluster: %s/%s .status.PD.Members count(%d) != %d",
-			ns, tcName, len(tc.Status.PD.Members), tc.Spec.PD.Replicas)
+		log.Logf("TidbCluster: %q .status.PD.Members count(%d) != %d", tcID, len(tc.Status.PD.Members), tc.Spec.PD.Replicas)
 		return false, nil
 	}
 	if pdSet.Status.ReadyReplicas != pdSet.Status.Replicas {
-		log.Logf("statefulset: %s/%s .status.ReadyReplicas(%d) != .status.Replicas(%d)",
-			ns, pdSetName, pdSet.Status.ReadyReplicas, pdSet.Status.Replicas)
+		log.Logf("StatefulSet: %q .status.ReadyReplicas(%d) != .status.Replicas(%d)", pdStsID, pdSet.Status.ReadyReplicas, pdSet.Status.Replicas)
 		return false, nil
 	}
 
 	c, found := getMemberContainer(oa.kubeCli, oa.tcStsGetter, ns, tc.Name, label.PDLabelVal)
 	if !found {
-		log.Logf("statefulset: %s/%s not found containers[name=pd] or pod %s-0",
-			ns, pdSetName, pdSetName)
+		log.Logf("StatefulSet: %q not found containers[name=pd] or pod %s-0", pdStsID, pdSetName)
 		return false, nil
 	}
 
 	if tc.PDImage() != c.Image {
-		log.Logf("statefulset: %s/%s .spec.template.spec.containers[name=pd].image(%s) != %s",
-			ns, pdSetName, c.Image, tc.PDImage())
+		log.Logf("StatefulSet: %q .spec.template.spec.containers[name=pd].image(%s) != %s", pdStsID, c.Image, tc.PDImage())
 		return false, nil
 	}
 
 	for _, member := range tc.Status.PD.Members {
 		if !member.Health {
-			log.Logf("tidbcluster: %s/%s pd member(%s/%s) is not health",
-				ns, tcName, member.ID, member.Name)
+			log.Logf("TidbCluster: %q pd member(%s/%s) is not health", tcID, member.ID, member.Name)
 			return false, nil
 		}
 	}
 
-	pdServiceName := controller.PDMemberName(tcName)
-	pdPeerServiceName := controller.PDPeerMemberName(tcName)
-	if _, err := oa.kubeCli.CoreV1().Services(ns).Get(pdServiceName, metav1.GetOptions{}); err != nil {
-		log.Logf("failed to get service: %s/%s", ns, pdServiceName)
+	if _, err := oa.kubeCli.CoreV1().Services(ns).Get(pdSetName, metav1.GetOptions{}); err != nil {
```

Comment:
It's confusing to use the set name here even if it has the same value as `controller.PDMemberName(tcName)`.

Reply:
fixed in next commits